### ISSUE TITLE
Setup hook for deployment with tools like this one, test deployment pipeline with dummy image.

### ISSUE THREAD

**Issue by Orhideous** [everypony/tabun#190]

**Comment by Orhideous** [comment]
@randomEP Any objections?

**Comment by randomEP** [comment]
Нет, против поднятия нового сервера (с чистой базой) для тестирования на тестовом домене ничего не имею.

**Comment by roman901** [comment]
Конкретно у себя использую другой флоу - дергаю с ci docker-compose с передачей -H "ssh://$SSH_USER@$SSH_HOST" - с авторизацией по ключу, естественно - и разворачиваю контейнеры сразу на хост.
